### PR TITLE
Shot at 889

### PR DESCRIPTION
**Pull request by xml-project**
First shot at #889 (only core specs). Changes on p:load and p:set-properties (step specs) are pending.

Does this work to clarifiy the relation and to solve the synch problem?

### CODE REVIEW

**Comment by xml-project** [comment]
Let's see what @ndw and @eriksiegel have to say.

@gimsieke 
> In the core spec, there needs to be additional clarity what the base-uri key of p:document/@document-properties does to the base URI of the read document (it should set the base URI to the value associtated with the base-uri key).

As we decided to define p:document completely via p:load, I think the clarification there should also contribute to the case you mentioned.

File: xproc/src/main/xml/specification.xml [hunk 1]
```diff
@@ -390,9 +390,9 @@ XProc pipeline as a map (<code>map(xs:QName, item()*)</code>).</termdef></para>
           </term>
           <listitem>
             <para>The value of the (optional) “<code>base-uri</code>” property identifies the base URI of the
-              document. If no such key is present, the document has no base URI. The processor is
-              responsible for assuring that the <code>base-uri</code> property matches the base URI
-              of each document produced on every output port.</para>
+              document. For XML document, HTML documents, and text documents, the value of “<code>base-uri</code>”
```
Comment (ndw) [inline xproc/src/main/xml/specification.xml]: "For XML document*s*, HTML ..."

**Comment by ndw** [comment]
Looks ok to me.

**Comment by xml-project** [comment]
Will supply the second half (p:load / p:set-properties) this afternoon.

**Pull request merged**